include/nccl_ofi_rdma.h

```diff
@@ -80,14 +81,22 @@ typedef enum nccl_net_ofi_rdma_req_type {
 	NCCL_OFI_RDMA_RECV_CONN_RESP,
 	/* Connect response message send request */
 	NCCL_OFI_RDMA_SEND_CONN_RESP,
+	/* Invalid type */
+	NCCL_OFI_RDMA_INVALID_TYPE,
 } nccl_net_ofi_rdma_req_type_t;
 
 enum nccl_ofi_rdma_msg_type {
-	NCCL_OFI_RDMA_MSG_CONN,
+	NCCL_OFI_RDMA_MSG_CONN = 0,
 	NCCL_OFI_RDMA_MSG_CONN_RESP,
 	NCCL_OFI_RDMA_MSG_CTRL,
-	NCCL_OFI_RDMA_MSG_EAGER
+	NCCL_OFI_RDMA_MSG_EAGER,
+	NCCL_OFI_RDMA_MSG_INVALID,
```

Comment:
The assumption here in the value aliasing of _MAX is that we will never serialize an invalid message type. We still have a lot of space here for new messages, so maybe not a concern that needs to be considered today, but want to call attention to this @bwbarrett

Reply:
I think this is reasonable.  We can always change it later.

Reply:
I ended up changing it, I didn't like this. Let's force it to be serializable and if we end up running out of space at some point, then we can get clever with this corner case.